audit.tf

```
@@ -87,6 +87,48 @@ resource "aws_sns_topic_policy" "monitor_iam_access_audit" {
   policy   = data.aws_iam_policy_document.monitor_iam_access_audit_topic.json
 }
 
+resource "aws_guardduty_detector" "audit" {
+  count    = var.aws_guardduty == true ? 1 : 0
+  provider = aws.audit
+}
+
+resource "aws_guardduty_member" "logging" {
+  count       = var.aws_guardduty == true ? 1 : 0
+  provider    = aws.audit
+  account_id  = aws_guardduty_detector.logging[0].account_id
+  detector_id = aws_guardduty_detector.audit[0].id
+  email       = local.aws_account_emails[aws_guardduty_detector.logging[0].account_id]
+  invite      = true
+
+  depends_on = [aws_guardduty_organization_admin_account.audit]
+  lifecycle {
+    ignore_changes = [email]
+  }
+}
+
+resource "aws_guardduty_member" "master" {
+  count       = var.aws_guardduty == true ? 1 : 0
+  provider    = aws.audit
+  account_id  = aws_guardduty_detector.master[0].account_id
+  detector_id = aws_guardduty_detector.audit[0].id
+  email       = local.aws_account_emails[aws_guardduty_detector.master[0].account_id]
+  invite      = true
+
+  depends_on = [aws_guardduty_organization_admin_account.audit]
```

As it's on a single line this should be attached to the rest of the block like this:

```suggestion
  depends_on  = [aws_guardduty_organization_admin_account.audit]
```

Or specified as a block:

```suggestion

  depends_on = [
    aws_guardduty_organization_admin_account.audit
  ]
  
```